packages/server/src/auth/auth.service.ts

```diff
@@ -67,6 +70,39 @@ export class AuthService {
     throw new HttpException('Unauthorized', HttpStatus.UNAUTHORIZED);
   }
 
+  /**
+   *
+   * @param projectId
+   * @param credential
+   * @returns JWT or 401 status code
+   */
+  async validateGoogle(projectId: string, credential: string): Promise<any> {
+    if (projectId == null || credential == null) {
+      throw new HttpException('Bad request: project id and credential required.', HttpStatus.BAD_REQUEST);
+    }
```

Comment:
Minor: you have validation thru decorators, so you should need to have it hear as well